src/aoWebWallet/wwwroot/css/app.css

```diff
@@ -427,6 +427,10 @@ body {
     background-color: rgba(16,16,67,1) !important;
 }
 
+.mud-list {
+    background-color: rgba(16,16,67,1) !important;
+}
+
```

Comment:
To make sure the autocomplete is not transparant. Does this look good to you @lopezi ?